src/pocketmine/entity/FloatingText.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace pocketmine\entity;
+
+use pocketmine\item\Item as ItemItem;
+use pocketmine\network\protocol\AddPlayerPacket;
+use pocketmine\Player;
+use pocketmine\utils\UUID;
+
+class FloatingText extends Entity {
+    const NETWORK_ID = 80;
```

Comment:
That's a bad idea. It will cause creating an createEntity() with 80 to create a FTP instead of an arrow.